### PR TITLE
Do not filter out ibc events

### PR DESCRIPTION
**Pull request by pr0n00gler**

### CODE REVIEW

File: x/wasm/keeper/msg_dispatcher.go [hunk 1]
```diff
@@ -188,12 +188,21 @@ func redactError(err error) error {
 	return fmt.Errorf("codespace: %s, code: %d", codespace, code)
 }
 
+func isIBCEvent(event sdk.Event) bool {
+	for _, attr := range event.Attributes {
+		if bytes.Contains(attr.Value, []byte("ibc_")) {
```
Comment (swelf19) [inline x/wasm/keeper/msg_dispatcher.go]: Are we expecting "ibc_" at any place of string? I mean why not `HasPreffix`. According to the hermes, he is subscribed only on the `module` key. May be leave a link to the hermes's line with events description?

File: x/wasm/keeper/msg_dispatcher_test.go [hunk 2]
```diff
@@ -295,9 +294,8 @@ func TestDispatchSubmessages(t *testing.T) {
 			},
 			expData:    nil,
 			expCommits: []bool{true},
-			expEvents:  []sdk.Event{sdk.NewEvent("message"), sdk.NewEvent("execute", sdk.NewAttribute("foo", "bar"))},
+			expEvents:  []sdk.Event{sdk.NewEvent("execute", sdk.NewAttribute("foo", "bar"))},
```
Comment (swelf19) [inline x/wasm/keeper/msg_dispatcher_test.go:297]: add a simple test case with an IBC related events?

File: x/wasm/keeper/msg_dispatcher.go (hunk 1 above)
Comment (mmikev) [reply x/wasm/keeper/msg_dispatcher.go]: Fixed

File: x/wasm/keeper/msg_dispatcher_test.go (hunk 2 above)
Comment (mmikev) [reply x/wasm/keeper/msg_dispatcher_test.go:297]: Fixed

**Review by swelf19 — APPROVED** [review]

**Review by zavgorodnii — APPROVED** [review]

**Pull request merged**